src/test/java/TC_DP194_50.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import steps.CategoryPageStep;
+import steps.CheckoutPageRegisterStep;
+import steps.HomePageStep;
+
+public class TC_DP194_50 extends CommonConditionTest{
+
+    String contactPageUrl = "http://34.121.117.87/index.php?route=checkout/success";
```

Comment:
We use as an expected URL constants in PageURL interface (constants package). You should have it (package) if you pulled last updates from div branch